pkg/controller/configurationpolicy/configurationpolicy_controller.go

```diff
@@ -1438,7 +1518,7 @@ func updateTemplate(
 	return false, false, "", false
 }
 
-// AppendCondition check and appends conditions
+// AppendCondition check and appends conditions to the policy status
```

Comment:
From what I can see, this only appends to the list if `len(conditions) == 0`, is that on purpose? I think the whole history of conditions of a Policy is preserved in the propagator on the hub, so I'm not sure if it's also needed here, but I wanted to check.